src/components/shared/Activity/index.tsx

```diff
@@ -186,22 +191,59 @@ class ActivityContent extends React.Component<{
   ownVideoElems: VideoWithPlaceholderView[] = [];
 
   public render() {
-    const { actor, description, body } = this.props.content;
+    const { actors, description, body } = this.props.content;
+    const actorsArray: [typeof RAHA_BASIC_INCOME_MEMBER] | Member[] =
```

Comment:
This is saying it could either be one `RAHA_BASIC_INCOME_MEMBER` or an array of real members? Is there a cleaner way to do that?